book/getting-started.rst

```diff
@@ -54,6 +44,17 @@ This command will bootstrap a new project in the directory ``my-project``.
                 - en
                 - de
 
+    Available languages are shown on `Crowdin`_.
+    Afterwards the languages had been activated run the following command:
```

Comment:
"Afterwards the languages have to be downloaded by running the following command:"